src/aws-cpp-sdk-core/source/internal/AWSHttpResourceClient.cpp

```diff
@@ -208,22 +208,27 @@ namespace Aws
 #endif
         }
 
-        EC2MetadataClient::EC2MetadataClient(const Aws::Client::ClientConfiguration::CredentialProviderConfiguration& credentialConfig,
-                                             const char* endpoint)
-            : AWSHttpResourceClient(
-                  [&credentialConfig]() -> ClientConfiguration{
-                    Aws::Client::ClientConfiguration clientConfig{credentialConfig.profile.c_str()};
-                    clientConfig.region = credentialConfig.region;
-                    clientConfig.credentialProviderConfig = credentialConfig;
-                    clientConfig.requestTimeoutMs = credentialConfig.imdsConfig.metadataServiceTimeout * 1000;
-                    clientConfig.retryStrategy = credentialConfig.imdsConfig.imdsRetryStrategy;
-                    return clientConfig;
-                  }(),
-                  EC2_METADATA_CLIENT_LOG_TAG),
-              m_endpoint(endpoint),
-              m_disableIMDS(credentialConfig.imdsConfig.disableImds),
-              m_tokenRequired(true),
-              m_disableIMDSV1(credentialConfig.imdsConfig.disableImdsV1) {
+    EC2MetadataClient::EC2MetadataClient(const Aws::Client::ClientConfiguration::CredentialProviderConfiguration& credentialConfig,
```

Comment:
nothing changed in this function please remove uneeded white space change from this